docs/source/Howtos/Turn-based-Combat-System.md

```diff
@@ -419,21 +420,21 @@ class CmdAttack(Command):
     def func(self):
         "Handle command"
         if not self.args:
-            self.caller.msg("Usage: attack <target>")            
+            self.caller.msg("Usage: attack <target>")
             return
         target = self.caller.search(self.args)
         if not target:
             return
         # set up combat
         if target.ndb.combat_handler:
             # target is already in combat - join it            
-            target.ndb.combat_handler.add_character(self.caller)
+            target.ndb.combat_handler.add_character_to_playable_list(self.caller)
             target.ndb.combat_handler.msg_all(f"{self.caller} joins combat!")
         else:
             # create a new combat handler
             chandler = create_script("combat_handler.CombatHandler")
-            chandler.add_character(self.caller)
-            chandler.add_character(target)
+            chandler.add_character_to_playable_list(self.caller)
```

Comment:
Same here, it doesn't make sense to rename this, especially since the combathandler's code was not updated to match (and it's unrelated to this anyway)